students/Ruohan/lesson04/kata.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+''' trigrams '''
+import string
+import random
+
+file_name = 'sherlock.txt'
+with open (file_name) as f_handle:
+# find the real content we need from the file
+    line_useful = ''
+    bool_flag = False
```

Comment:
try to use variable names that mean something, like "valid_line" or "good_data" or whatever, the fact that its a boolean flag doesn't tell us anything about what its meaning is.